src/elements/core/mixins/form-associated-mixin.ts

```diff
@@ -192,12 +192,15 @@ export const SbbFormAssociatedMixin = <T extends Constructor<LitElement>>(
       reason: FormRestoreReason,
     ): void;
 
-    /** Should be called when form value is changed. */
-    protected updateFormValue(): void {
-      this.internals.setFormValue(this.value);
-    }
+    /**
+     * Should be called when form value is changed.
+     * Adapts and sets the formValue in the supported format (string | FormData | File | null)
+     * https://developer.mozilla.org/en-US/docs/Web/API/ElementInternals/setFormValue
+     */
+    protected abstract updateFormValue(): void;
```

Comment:
If you mark this as abstract, the mixin type above should also mark it as abstract